src/components/TimerComponent.jsx

```diff
@@ -56,22 +56,23 @@ const TimerComponent = () => {
   return (
     <div style={{ padding: '10px', border: '1px solid black' }}>
       <div>
-        <button type='button' onClick={handleIncrease}>
+        <button type="button" onClick={handleIncrease}>
```

Comment:
This is another example of ESLint consistancies. When to use single and double quotes are part of it's magic.

Reply:
Yeah, I remember.